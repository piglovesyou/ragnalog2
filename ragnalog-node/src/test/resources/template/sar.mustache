in:
  type: sar
  file_path: {{input_file}}
  sar_options: {{sar.sar_options}}
  sadf_paths:
    {{#sar.sadf_paths}}
    - {{_1}} : {{_2}}
    {{/sar.sadf_paths}}
  exclude_fields:
    {{#sar.exclude_fields}}
    - {{.}}
{{/sar.exclude_fields}}
filters:
  - type: ragnalog
    columns:
      extra: {{extra}}
out:
  type: elasticsearch
  nodes:
    - {host: {{elasticsearch.host}}, port: {{elasticsearch.port}} }
  cluster_name: {{elasticsearch.cluster_name}}
  bulk_actions: 30000
  index: {{index_name}}
  index_type: sar