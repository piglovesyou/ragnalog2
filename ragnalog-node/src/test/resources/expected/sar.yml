in:
  type: sar
  file_path: /tmp/ragnalog/sample.sar
  sar_options: -A -U
  sadf_paths:
    - '9': /opt/sysstat-9.0.6/sadf
    - '10': /opt/sysstat-10.0.5/sadf
    - '11': /opt/sysstat-11.0.7/sadf
  exclude_fields:
    - interrupts.*
filters:
  - type: ragnalog
    columns:
      extra: ap1
out:
  type: elasticsearch
  nodes:
    - {host: localhost, port: 9300 }
  cluster_name: ragnalog2.elasticsearch
  bulk_actions: 30000
  index: ragnalog-test-sar
  index_type: sar